connectors/http-json/element-templates/http-json-connector.json

```diff
@@ -74,6 +74,10 @@
         {
           "name": "OAuth 2.0",
           "value": "oauth-client-credentials-flow"
+        },
+        {
+          "name": "Custom Authentication",
```

Comment:
`Custom Authentication` is too generic name - it's applicable for every non-standard types of authentication.
---
Consider something like `Credentials in body`.
---
Also, we don't append word `Authentication` in type because it's already mentioned in the field.

What do you think @Oleksiivanov @markfarkas-camunda? Any better suggestions for naming?

Reply:
I am not in favor of any. `Custom`, `specific`, `unregular`, etc do not explain what and how exactly it is implemented and sending.

When we say `Basic` (authentication) we know that we refer to specific RFC of basic authentication which is always `base64(login+:+password)` function. When we say `OAuth 2.0` we always know the mechanism as well. But when we say `Custom` I have no idea what is it supposed to mean.

If in the next iteration we find another mechanism of authentication, say, for BluePrism how are we going to call it? That's why we should explicitly name the authenctication type to let user understand what's going on without looking into code.

My suggestion is `Credentials in body` with description that `Login and password will be sent in body as JSON object prior to calling resource endpoint blah-blah`. But since I am a bit out of context, maybe there's better name.